### ISSUE TITLE
How to get what is the progress of archiving?

### ISSUE THREAD

**Issue by mareknovotny** [Brummolix/AutoarchiveReloaded#55]
If I click on AutoArchive... the status or progress is not shown anywhere, is it possible to see it somewhere. Even error log dialog show only sporadically anything about status. So i am not sure when the archiving process stops

**Comment by Brummolix** [comment]
The archive collects all messages to be archived and then tells Thunderbird to archive them.
For archiving you can see some (Thunderbird) status under:
Hamburger menu (the button with the 3 lines at the top right)\Extras\activity

**Comment by mareknovotny** [comment]
That's right I saw one message about starting archivation, but it doesn't continue or notify or log any status if it has ended or still in progress. If there is no such message logged how the addon is able to identify to not call archivation multiple times for the same message and have in parallel more than necessary processes?
Why I am asking is that i got few times a message that operation on a folder is not possible as it is used by another operation in progress.

**Comment by Brummolix** [comment]
> how the addon is able to identify to not call archivation multiple times for the same message and have in parallel more than necessary processes

Indeed the addon does not now. If you start the autoarchive process twice it may lead to errors.
The message you got may be from such 2 archivings. But it could also be from other Thunderbird processes, for example if Thunderbird asks to compact folders you are also not able to do other message functions in such a folder.
